assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -65,30 +75,50 @@ const baseInitialState = {
 
 const baseActions = {
 	/**
-	 * Sets user input settings.
+	 * Gets cached user input settings and save them to the data store.
 	 *
-	 * @since 1.19.0
+	 * @since n.e.x.t
 	 *
-	 * @param {Object} values User input settings.
-	 * @return {Object} Redux-style action.
+	 * @return {Object} Cached user input answer values.
 	 */
-	setUserInputSettings( values ) {
-		return {
-			type: SET_USER_INPUT_SETTINGS,
-			payload: { values },
```

Comment:
Note: this was _not_ working as-expected in my usage… using it to set all values resulted in bugs. It's possible the API wasn't clear/documented incorrectly, but more-to-the-point: we don't use this anywhere in the code and it wasn't even useful for this PR really, so I've removed it as:

1. we weren't using it
2. it wasn't working as-expected

The second point matters more, as it led me down a bit of a rabbit hole wondering why the code didn't work. Upon switching to `setUserInputSetting` it worked immediately, so I figured it was better to remove unused, unexpected code.